qemu/tests/cfg/balloon_hotplug.cfg

```diff
@@ -16,6 +16,9 @@
     test_tags = "evict enlarge"
     balloon_type_evict = evict
     balloon_type_enlarge = enlarge
+    balloon_device = virtio-balloon-pci
+    s390x:
+        balloon_device = virtio-balloon-ccw
```

Comment:
Minor typo in the commit message: `s/ballon/balloon/`

Reply:
Fixing and updating here in v2